addons/web/models/ir_http.py

```diff
@@ -28,7 +28,7 @@
 Multiple debug modes can be activated simultaneously, separated with a
 comma (eg: 'tests, assets').
 """
-ALLOWED_DEBUG_MODES = ['', '1', 'assets', 'tests']
+ALLOWED_DEBUG_MODES = ['', '1', 'assets', 'tests', 'nocache']
```

Comment:
I propose `noqwebcache` so that people are not confused with the browser or worker cache